packages/shared/src/tms/proj.ts

```diff
@@ -0,0 +1,12 @@
+import * as projImport from 'proj4';
```

Comment:
There is no shared lib that has proj in both frontend and backend code hence the copy pasta